donor-registry/src/assets/config/ui-config/forms.json

```diff
@@ -111,6 +111,7 @@
                         "pattern": "^[a-zA-Z\\s-']+$",
                         "message": "Enter a valid name"
                       },
+                      "enableField": true,
```

Comment:
@suvarnakale for edit form it will worl as expected?

Reply:
Yeah... For edit form it is already done.